### PR TITLE
Fix requires_ansible check when no meta/runtime.yml

### PR DESCRIPTION
**Pull request by awcrosby**
Fail import when there is no meta/runtime.yml

Fixes a missed case from #122 and AAH-538

No-Issue

### CODE REVIEW

File: tests/unit/test_collection.py [hunk 1]
```diff
@@ -77,7 +77,7 @@
   "name": "FILES.json",
   "ftype": "file",
   "chksum_type": "sha256",
-  "chksum_sha256": "dc90402feea54f479780e067cba748559cb01bff52e6724a15264b9a55e8f000",
+  "chksum_sha256": "0c2e9514f54aaeda0605d8c9a60e71cab3e577113b58ecc65b839d2d1b86c216",
```
Comment (alikins) [inline tests/unit/test_collection.py:80]: _<general comment on existing code, not a requested change...>_

Updating the tests to calculate this on the fly might be useful in the future. I didn't do it in an attempt to stay focused on the feature. Maybe it's better as is for explicitness, I'm not sure. It is a tiny bit annoying to update though.

Comment (awcrosby) [reply tests/unit/test_collection.py:80]: Haven't had to update this much so should be ok, and wasn't a bad update

File: galaxy_importer/loaders.py [hunk 2]
```diff
@@ -193,7 +193,7 @@ def _load(self):
 
     def get_requires_ansible(self):
         if not self.data:
-            return
+            raise exc.RuntimeFileError("'requires_ansible' in meta/runtime.yml is mandatory")
```
Comment (alikins) [inline galaxy_importer/loaders.py:196]: Probably fine. Maybe tweak the error msg to indicate the contents of meta/runtime.yml was empty?

**Review by alikins — APPROVED** [review]
LGTM. Small suggestion for updating the errmsg for the empty (or False-y) contents of runtime.yml, but that is not a blocker.

File: galaxy_importer/loaders.py (hunk 2 above)
Comment (awcrosby) [reply galaxy_importer/loaders.py]: updated

**Comment by felixfontein** [comment]
@awcrosby @alikins neither `meta/runtime.yml` nor `requires_ansible` are mandatory for collections. Why do you require them?

**Pull request merged**